tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -178,17 +178,18 @@ def run(self):
                                     connected=peers['peer%s.connected' % index]
                                 )
                             try:
-                                stored_peer_status = NS._int.client.read(
-                                    "clusters/%s/Peers/%s/connected" % (
-                                        NS.tendrl_context.integration_id,
-                                        peers['peer%s.uuid' % index]
-                                    )
+                                stored_peer_status = etcd_utils.read(
```

Comment:
NS.tendrl.objects.ClusterPeer(...).load() ?

Reply:
This block of code will be moved to  GlusterPeer.on_change function soon